src/Storage/Storage.Management/help/Set-AzStorageAccountManagementPolicy.md

```diff
@@ -0,0 +1,404 @@
+
+## SYNTAX
+
+### AccountNamePolicyRule (Default)
+```
+Set-AzStorageAccountManagementPolicy [-ResourceGroupName] <String> [-StorageAccountName] <String>
+ -Rule <PSManagementPolicyRule[]> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### AccountNamePolicyObject
+```
+Set-AzStorageAccountManagementPolicy [-ResourceGroupName] <String> [-StorageAccountName] <String>
+ -Policy <PSManagementPolicy> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### AccountObjectPolicyRule
+```
+Set-AzStorageAccountManagementPolicy -StorageAccount <PSStorageAccount> -Rule <PSManagementPolicyRule[]>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### AccountObjectPolicyObject
+```
+Set-AzStorageAccountManagementPolicy -StorageAccount <PSStorageAccount> -Policy <PSManagementPolicy>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### AccountResourceIdPolicyRule
+```
+Set-AzStorageAccountManagementPolicy [-StorageAccountResourceId] <String> -Rule <PSManagementPolicyRule[]>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### AccountResourceIdPolicyObject
+```
+Set-AzStorageAccountManagementPolicy [-StorageAccountResourceId] <String> -Policy <PSManagementPolicy>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Set-AzStorageAccountManagementPolicy** cmdlet creates or modifies the management policy of an Azure Storage account.
+
+## EXAMPLES
+
+### Example 1: Create or update the management policy of a Storage account with ManagementPolicy rule objects.
+```
+PS C:\>$action1 = Add-AzStorageAccountManagementPolicyAction -BaseBlobAction Delete -daysAfterModificationGreaterThan 100
+PS C:\>$action1 = Add-AzStorageAccountManagementPolicyAction -InputObject $action1 -BaseBlobAction TierToArchive -daysAfterModificationGreaterThan 50
+PS C:\>$action1 = Add-AzStorageAccountManagementPolicyAction -InputObject $action1 -BaseBlobAction TierToCool -daysAfterModificationGreaterThan 30
+PS C:\>$action1 = Add-AzStorageAccountManagementPolicyAction -InputObject $action1 -SnapshotAction Delete -daysAfterCreationGreaterThan 100
+PS C:\>$filter1 = New-AzStorageAccountManagementPolicyFilter -PrefixMatch ab,cd 
+PS C:\>$rule1 = New-AzStorageAccountManagementPolicyRule -Name Test -Action $action1 -Filter $filter1
+
+PS C:\>$action2 = Add-AzStorageAccountManagementPolicyAction -BaseBlobAction Delete -daysAfterModificationGreaterThan 100
+PS C:\>$filter2 = New-AzStorageAccountManagementPolicyFilter
+PS C:\>$rule2 = New-AzStorageAccountManagementPolicyRule -Name Test2 -Action $action2 -Filter $filter2
+
+PS C:\>Set-AzStorageAccountManagementPolicy -ResourceGroupName "myresourcegroup" -AccountName "mystorageaccount" -Rule $rule1,$rule2
+
+
+ResourceGroupName  : myresourcegroup
```

Comment:
The formatted result should be in table format - You need to show the table formatted output

Reply:
@markcowl 
I still prefer list view than table view, since:
1. The output type has several levels, can't show completely in a table view. With ListView, it can show completely, and easy for customer to check the policy/rule created/updated.
2. Generally, customer will handle one policy/rule/ActionGroup with a command, since one account only has one policy, and 1 rule only has 1 ActionGroup. So the table view don't have much advantage there than list view. 

I have filed another PR to change the list view to table view in https://github.com/Azure/azure-powershell/pull/8791.

If you agree we should keep using listview, please merge this PR.
If you still think table view is better, you can merge https://github.com/Azure/azure-powershell/pull/8791

If there's anything still need to change the PRs before merge, let's have a meeting tomorrow to track the change quickly and make it merge in time.

Reply:
Resolved offline - we will look at formats in the future